smartdashboard/view_builders.py

```diff
@@ -87,6 +87,16 @@ def exp_builder(manifest: Manifest) -> ExperimentView:
     st.write("Path: " + manifest.experiment.path)
     st.write("Launcher: " + manifest.experiment.launcher)
 
+    with st.expander(label="Logs"):
+        col1, col2 = st.columns([6, 6])
+        with col1:
+            st.write("Output")
+            view.out_logs_element = st.code(view.out_logs, language=None)
```

Comment:
have we tried any of the languages supported to see if these logs render nicely?

there is a language `"log"` that might be a good place to start: https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/AVAILABLE_LANGUAGES_PRISM.MD

Reply:
We have a semi-fixed log formatting string for the experiment logs. If you wanted to reformat the logs as dictionaries using the elements, we could get really, really nice looking logs that can expand/contract or have a default view that is truncated.

I still have some time to adjust the log format before release. If I change it to be a real CSV style format it would make your life easy and you'd be able to have this fancy log rendering do json or anything else.

Reply:
switching to `language = "log"` is where we're going to start with this, but making logs fancier in the future for all entities is something we should absolutely do in the future!